### PR TITLE
Изменено «пиратский сервер» → «частный сервер»

### PR DESCRIPTION
**Pull request by Simplaf234-ghost**
Calling them "pirate's servers" is common in Russian, but "private" actually means "частный".

### CODE REVIEW

**Comment by FrancescoBorzi** [comment]
@Viste @Winfidonarleyan can you please review ?

**Comment by Viste** [comment]
@FrancescoBorzi i think better use familiar name then same as in eng variant :) no one use private server in russian all use pirate

**Comment by Simplaf234-ghost** [comment]
I don’t think it is correct to call them "pirate's servers" even if everyone calls them so. Use "_unofficial servers_" (_неофициальные серверы_) then. Readers will understand it.

This word is [disliked](https://www.gnu.org/philosophy/words-to-avoid.html#Piracy) by GNU.

I also think this name is incorrect because of differences between _piracy_ (copyright infringement) and current private servers. Here are reasons for this and they are also reasons why I hate _all_ of WoW private servers and why I quit WoW permanently (most likely these issues will never be fixed and I don’t want to play alone on my local server):

tl;dr IIUC, _pirate_ server is about playing the game **as‐is** without paying for it, but IRL private servers are about playing custom WoW‐based game on fixed patch.

IIUC, _pirate_ server means it’s used to play game **as‐is** for free if it is meant to be paid for. In case of WoW, it must be as _blizzlike_ as possible (the only exception is it must be free‐to‐play). By _blizzlike_ I also mean:

- blizzlike progression (from 1.1.0 to last WoW patch which number we don’t know yet with correct delays between patches)
- blizzlike realms (all locales, not just few any‐locale realms)
- blizzlike ToS and GM behavior
- blizzlike environment (in 3.3 they implemented cross‐game _RealID_ and _Battle.net_ account support so _pirate_ server developers should work together on this)
- etc.

Of course such server will be immediately shut down by Blizzard Entertainment, but WoW is using TCP and we can use Tor for it. Even if one such server is shut down with its staff arrested, another one will appear, like there were Silk Road, then Silk Road 2, then third one.

IRL, most likely no such server exists.
Private servers have serious differences from retail ones:

- Almost all private servers are stuck on latest patch of some expansion. Some of them use non‐blizzlike static client patch progression or live progression in case latest expansion patch is not yet released by Blizzard Entertainment. And when progression ends, they start implementing non‐blizzlike stuff instead of progressing to next major patch. Some servers which don’t are sometimes called "blizzlike" but they aren’t due to lack of _blizzlike_ major patch progression.
- Some servers are intentionally non‐blizzlike (like increased rates or imbalanced donation items).
- Most private servers have single or few any‐locale realms (because there are way less players than on retail).
- They have their own ToS and GM hiring requirements.

And even emulator developers refused to implement _Battle.net accounts/RealID_ because it depends on some public‐key cryptographic system where private key is not known (in case of true _pirate_ server they would patch the client and implement it using their own key).

**Comment by FrancescoBorzi** [comment]
I'm really not sure what to do with this as I don't understand Russian. Any third opinion?

**Comment by TheSCREWEDSoftware** [comment]
I know it's few years later to this @FrancescoBorzi but I agree in keeping in "Pirate Server" because is what's normally called in russian. Last time i heard in wow in russian refered to a private server was always refered to a premium realm part of a "pirate server".

The translation should reflect the original post but with the changes from references and differences in cultures to match the most commun / majority of the people of that language, this PR in my opnion could be closed
